client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat

```diff
@@ -79,9 +53,9 @@ goto :eof
   cd %AVRO_SRC%\lang\c++\build.win
 
   if %BUILD_PLATFORM% == x86 (
-    cmake -DCMAKE_INSTALL_PREFIX:PATH=%AVRO_ROOT% -G "Visual Studio %MSVC_VERSION%"  ..
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=%ROOT_PATH% -G "Visual Studio %MSVC_VERSION%"  ..
```

Comment:
If the current build is not broken, I would prefer separate variables for different libraries than a single `ROOT_PATH`